src/test/java/com/ddudu/following/controller/FollowingControllerTest.java

```diff
@@ -275,4 +278,56 @@ void setUp() {
 
   }
 
+  @Nested
+  class DELETE_팔로잉_삭제_API_테스트 {
+
+    long randomId;
+    String token;
+
+    @BeforeEach
+    void setUp() {
+      randomId = faker.random()
+          .nextLong();
+      token = createBearerToken(randomId);
+    }
+
+    @Test
+    void 팔로잉_삭제를_성공하고_No_Content를_반환한다() throws Exception {
```

Comment:
```suggestion
    void 팔로잉_삭제를_성공하고_204_No_Content를_반환한다() throws Exception {
```

아래 403_Forbidden 처럼 코드도 넣어서 통일시켜주는 건 어떻게 생각하시나요?